bvm/ballerina-logging/src/main/java/org/ballerinalang/logging/BLogManager.java

```diff
@@ -86,6 +89,15 @@ public void readConfiguration(InputStream ins) throws IOException, SecurityExcep
             properties.setProperty((String) k, val);
         });
 
+        String breLogFileName = properties.getProperty(DEFAULT_LOG_FILE_HANDLER_PATTERN);
```

Comment:
Do we need to handle this? The `logging.properties` file is in the same jar as this file. So it's unlikely that this would be unavailable/empty right?